www_src/lib/render.jsx

```diff
@@ -1,9 +1,12 @@
 var React = require('react');
+
 var Spindicator = require('../components/spindicator/spindicator.jsx');
 var ModalConfirm = require('../components/modal-confirm/modal-confirm.jsx');
 var ModalSwitch = require('../components/modal-switch/modal-switch.jsx');
 var Snackbar = require('../components/snackbar/snackbar.jsx');
 
+var platform = require('./platform');
```

Comment:
Is there a reason that this is an explicit include, and elsewhere Platform is a global?